server/utils/referralUtils.test.ts

```diff
@@ -60,7 +60,7 @@ describe('ReferralUtils', () => {
 
   describe('taskListSections', () => {
     it('returns task list sections for a given referral', () => {
-      const referral = referralFactory.build()
+      const referral = referralFactory.build({ reason: undefined })
```

Comment:
Per earlier comment, I think this should be the default for the factory, so if that changes, this change can probably be reverted

Reply:
Let's use a trait to represent a `referral_started` Referral to be more explicit here. I think we can do that as a separate tidyup PR rather than blocking this one.